client/src/sim.rs

```diff
@@ -437,7 +437,15 @@ impl Sim {
     }
 
     pub fn view(&self) -> Position {
-        self.local_character_controller.oriented_position()
+        let mut pos = self.local_character_controller.position();
```

Comment:
I believe you should be able to simplify things by using `oriented_position()` here, which would remove the need to use `* self.local_character_controller.orientation().to_homogeneous()`.

Because `oriented_position()` is used as the argument to `get_relative_up`, the coordinate system should be correct, making up for the change in the order of multiplication.

Reply:
Right you are. https://github.com/Ralith/hypermine/pull/346.